python/kfserving/setup.py

```diff
@@ -12,32 +12,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from setuptools import setup, find_packages
+import setuptools
 
-tests_require = [
-    'pytest',
-    'pytest-tornasync',
-    'mypy'
+REQUIRES = [
+    'certifi >= 14.05.14',
+    'six >= 1.10',
+    'python_dateutil >= 2.5.3',
+    'setuptools >= 21.0.0',
+    'urllib3 >= 1.15.1',
+    'kubernetes >= 9.0.0',
+    'tornado >= 1.4.1',
+    'argparse >= 1.4.0',
+    'minio >= 4.0.9',
+    'google-cloud-storage >= 1.16.0',
+    'azure-storage-blob >= 2.0.1',
+    'numpy',
 ]
 
-setup(
-    name='kfserver',
-    version='0.1.0',
+setuptools.setup(
+    name='kfserving',
+    version='0.0.2',
+    author="Kubeflow Authors",
     author_email='ellisbigelow@google.com',
     license='../../LICENSE.txt',
-    url='https://github.com/kubeflow/kfserving/python/kfserving/kfserving',
-    description='Model Server for arbitrary python ML frameworks.',
+    description="Python SDK for KFServing",
+    url="https://github.com/kubeflow/kfserving/python/kfserving",
     long_description=open('README.md').read(),
     python_requires='>3.4',
-    packages=find_packages("kfserving"),
-    install_requires=[
-        "tornado >= 1.4.1",
-        "argparse >= 1.4.0",
-        "minio >= 4.0.9",
-        "google-cloud-storage >= 1.16.0",
-        "azure-storage-blob >= 2.0.1",
-        "numpy"
+    packages=[
+        'kfserving',
+        #'kfserving.api',
+        #'kfserving.constants',
+        'kfserving.models',
+        #'kfserving.utils'
     ],
-    tests_require=tests_require,
-    extras_require={'test': tests_require}
+    package_data={},
+    include_package_data=False,
+    zip_safe=False,
+    classifiers=[
+        'Intended Audience :: Developers',
+        'Intended Audience :: Education',
+        'Intended Audience :: Science/Research',
```

Comment:
'Intended Audience :: Developers',
'Intended Audience :: Academists',
'Intended Audience :: Scientists/Researchers',

Reply:
Updated. Thanks!

Reply:
Rollbacked, since the updated ones are not valid. @animeshsingh 

```
Error: 'Intended Audience :: Academists' is not a valid
Error: 'Intended Audience :: Scientists/Researchers' is not a valid
```